src/com/jsonnetplugin/JsonnetFileTypeFactory.java

```diff
@@ -6,34 +6,6 @@
 public class JsonnetFileTypeFactory extends FileTypeFactory {
     @Override
     public void createFileTypes(@NotNull FileTypeConsumer fileTypeConsumer) {
-        fileTypeConsumer.consume(
-                JsonnetFileType.INSTANCE,
-                new FileNameMatcher(){
-
-                    @Override
-                    public boolean accept(@NotNull String fileName) {
-                        return fileName.endsWith(".jsonnet");
-                    }
-
-                    @NotNull
-                    @Override
-                    public String getPresentableString() {
-                        return ".jsonnet";
-                    }
-                },
-                new FileNameMatcher(){
-
-                    @Override
-                    public boolean accept(@NotNull String fileName) {
-                        return fileName.endsWith(".jsonnet.TEMPLATE");
```

Comment:
Sorry I'm just coming to look at this; wouldn't this change mean `.jsonnet.TEMPLATE` extension is no longer supported? That's the naming convention we use internally

Reply:
Yes, it's not supported anymore. Why don't you use [the one recommended by Jsonnet, "libsonnet"](https://jsonnet.org/learning/tutorial.html#imports)?

> Files designed for import by convention end with .libsonnet

I haven't seen "jsonnet.TEMPLATE" anywhere else.